### ISSUE TITLE
Tutorial on using template types in the Patterns API

### ISSUE THREAD

**Issue by justintadlock** [WordPress/developer-blog-content#65]
This would be a new tutorial that covers using the `templateTypes` option in the Patterns API.  It is currently awaiting the UI component: https://github.com/WordPress/gutenberg/issues/42325

Related: locking to template types: https://github.com/WordPress/gutenberg/issues/47208

### Discussed in https://github.com/WordPress/developer-blog-content/discussions/58

<div type='discussions-op-text'>

<sup>Originally posted by **annezazu** January 16, 2023</sup>
Basically create a how to/application style post around using this new functionality: https://github.com/WordPress/gutenberg/pull/45814</div>

**Comment by annezazu** [comment]
Noting this PR too to keep an eye out for: https://github.com/WordPress/gutenberg/pull/47322

**Comment by justintadlock** [comment]
This post might also have some crossover with the [multiple homepage options](https://github.com/WordPress/developer-blog-content/discussions/66) idea. Since I was first assigned this post, I've been trying to think of interesting use cases, and I believe offering multiple patterns that are tied to the "front page" template type is worth exploring.  @annezazu's response on that discussion was a bit of a "light bulb" moment for me, so I'm mostly just noting this so I don't forget.

**Comment by justintadlock** [comment]
It looks like this landed in Gutenberg 15.5 RC1 (https://github.com/WordPress/gutenberg/pull/47322), so I'll begin work on this in the next week or so. That way, we'll have the content ready soon after the release.

**Comment by justintadlock** [comment]
I have started working on this in a Google Doc: https://docs.google.com/document/d/1NCT7Cb_0UJnuinQsocNWFc8Ua7j91v_FJ88IPurhDUs/edit?usp=sharing

**Comment by annezazu** [comment]
This is already looking great, Justin :)

**Comment by justintadlock** [comment]
This [first draft doc](https://docs.google.com/document/d/1NCT7Cb_0UJnuinQsocNWFc8Ua7j91v_FJ88IPurhDUs/edit#) is now ready for review.

**Comment by marybaum** [comment]
I’ll do a drive-by in the morning before 🎾

**Comment by marybaum** [comment]
Done! Now off to 🎾

**Comment by annezazu** [comment]
Looks good! Had one minor comment.

**Comment by justintadlock** [comment]
Thanks, @marybaum and @annezazu!

**Comment by justintadlock** [comment]
Published here: https://developer.wordpress.org/news/2023/04/using-template-patterns-to-build-multiple-homepage-designs/
